Pow_max: -54
EngTank_min: 1
EngTank_init: 10
Tankgain: 100
cbf_rate: 23
KE_max: 0.45
K_transl: 500

